Formula/neovim.rb

```diff
@@ -0,0 +1,72 @@
+class Neovim < Formula
+  desc "Ambitious Vim-fork focused on extensibility and agility"
+  homepage "https://neovim.io"
+
+  stable do
+    url "https://github.com/neovim/neovim/archive/v0.2.0.tar.gz"
+    sha256 "72e263f9d23fe60403d53a52d4c95026b0be428c1b9c02b80ab55166ea3f62b5"
+  end
+
+  head do
+    url "https://github.com/neovim/neovim.git", :shallow => false
+  end
+
+  depends_on "cmake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "jemalloc"
+  depends_on "lua@5.1"
+  depends_on "luajit"
+  depends_on "libuv"
+  depends_on "msgpack"
+  depends_on "unibilium"
+  depends_on "libtermkey"
+  depends_on "libvterm"
+  depends_on "gettext"
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
+
+  resource "mpack" do
+    url "https://luarocks.org/manifests/tarruda/mpack-1.0.6-0.src.rock", :using => :nounzip
+    sha256 "9068d9d3f407c72a7ea18bc270b0fa90aad60a2f3099fa23d5902dd71ea4cd5f"
+  end
+
+  resource "lpeg" do
+    url "https://luarocks.org/manifests/gvvaughan/lpeg-1.0.1-1.src.rock", :using => :nounzip
+    sha256 "149be31e0155c4694f77ea7264d9b398dd134eca0d00ff03358d91a6cfb2ea9d"
+  end
+
+  def install
+    resources.each do |r|
+      r.stage(buildpath/"deps-build/build/src/#{r.name}")
+    end
+
+    ENV.prepend "LUA_PATH", "#{buildpath}/deps-build/share/lua/5.1/?.lua"
+    ENV.prepend "LUA_CPATH", "#{buildpath}/deps-build/lib/lua/5.1/?.so"
+
+    cd "deps-build" do
+      system "luarocks-5.1", "build", "build/src/mpack/mpack-1.0.6-0.src.rock", "--tree=."
+      system "luarocks-5.1", "build", "build/src/lpeg/lpeg-1.0.1-1.src.rock", "--tree=."
+      system "cmake", "../third-party",
+            "-DUSE_BUNDLED=OFF",
+            *std_cmake_args
+
+      # Deparallelize required since LuaRocks 2.4.1 for reasons unknown
```

Comment:
I just realised that the luarock version that's in the lua formulas is 2.3.0 so the issue wouldn't happen at the moment. If those Formulas are updated then we might start seeing the issue unless it works now for some reason.